model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
```

Comment:
If we are not going to save plaintext then maybe we can create a salt field here.